frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -7,7 +7,8 @@ pods:
     placement: "{{PLACEMENT_CONSTRAINT}}"
     {{#ENABLE_VIRTUAL_NETWORK}}
     networks:
-      dcos:
+      {{VIRTUAL_NETWORK_NAME}}:
```

Comment:
should this be wrapped in a check for non-empty `{{VIRTUAL_NETWORK_NAME}}`? something like:
```
{{#VIRTUAL_NETWORK_NAME}}
  {{VIRTUAL_NETWORK_NAME}}:
    labels: {{CNI_PLUGIN_LABELS}}
{{/VIRTUAL_NETWORK_NAME}}
```
(here and below)